rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -137,10 +137,10 @@ object Runtime {
     val errorLog                                  = new ErrorLog()
     implicit val ft: FunctorTell[Task, Throwable] = errorLog
 
-    def dispatchTableCreator(space: RhoISpace, dispatcher: RhoDispatch): RhoDispatchMap = {
+    def dispatchTableCreator(space: RhoISpace,
+                             dispatcher: RhoDispatch,
+                             registry: Registry): RhoDispatchMap = {
```

Comment:
I like that you're unhardcoding the registry creation here.

Reply:
Maybe `dispatchTableCreator` should be `createDispatchTable`?

Reply:
I did it on purpose. See my previous comment.

Reply:
My point as a rename would be nice to have :confused:

Reply:
Feel free to resolve this thread without the rename